Bottom: 1324d56688023968aa7f4e3f852a9482467e62c5
Top:    984947db6ac64bcf1c3ee6645cac6844f69cb3f1
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-07 11:57:16 -0500

Refresh of add-docker-option

---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index 79bbd612..74c6785b 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -52,6 +52,7 @@
     - build
     - testssh
     - buildbot
+    - nexus
     tags:
     - docker_networking
   - name: Build the openssl Docker volume
